### PR TITLE
added the buildconfig, so it can be used by Jenkinsfile

### PR DESCRIPTION
**Pull request by goern**

### CODE REVIEW

File: openshift/buildConfig-template.yaml [hunk 1]
```diff
@@ -0,0 +1,69 @@
...
+  displayName: Git Reference
+  required: true
+  name: GITHUB_REF
+  value: 'master'
+
+- description: Tag of the output ImageStream the resulting container image should go to
+  displayName: ImageStream Tag
+  required: true
+  name: IMAGE_STREAM_TAG
+  value: 'latest'
```
Comment (fridex) [inline openshift/buildConfig-template.yaml:67]: Some IDEs/text editors are just broken :smiley:

Comment (goern) [reply openshift/buildConfig-template.yaml:67]: :/

Comment (goern) [reply openshift/buildConfig-template.yaml:67]: it's Visual Studio Code, from our friends at Microsoft!

**Pull request merged**